crates/graph/src/range/elem/elem_enum/range_elem.rs

```diff
@@ -0,0 +1,586 @@
+use crate::elem::MinMaxed;
+use crate::{
+    nodes::{Concrete, ContextVarNode},
+    range::elem::{collapse, Elem, MaybeCollapsed, RangeElem},
+    GraphBackend, GraphError,
+};
+
+use shared::{NodeIdx, RangeArenaIdx};
+
+use shared::NodeIdx;
```

Comment:
fix compile error on import dupe
```suggestion
use shared::{NodeIdx, RangeArenaIdx};
```